tests/test/check/main.fmf

```diff
@@ -16,6 +16,12 @@ tier: 2
       - provision-local
       - provision-virtual
 
+    adjust+:
+      - when: provision_how == local
```

Comment:
```suggestion
      - when: provision-how == local
```

Would you mind of we call the new `context` adjust dimension `provision-how`. The naming would be consistent with other lower-case keys we're using across the config.